modules/beta-private-cluster-update-variant/cluster.tf

```diff
@@ -429,6 +429,15 @@ resource "google_container_node_pool" "pools" {
     }
   }
 
+  dynamic "network_config" {
+    for_each = lookup(each.value, "network_config", false) ? [each.value] : []
+    content {
+      create_pod_range    = lookup(network_config.value, "create_pod_range", false)
```

Comment:
I'm not sure we want to dynamically create the range on demand, since that's not very declarative.

Reply:
Cool, ill remove both create_pod_range and cidr vars.

Reply:
fixed

Reply:
Hi fellows, why not giving the possibility to customize the CIDR if set? Looks like pod_ipv4_cidr_block can not be set so it is set to a default. How can we set this to some CIDR range of our choosing? Thanks!